pkg/client/devops/jenkins/options.go

```diff
@@ -87,4 +93,7 @@ func (s *Options) AddFlags(fs *pflag.FlagSet, c *Options) {
 
 	fs.StringVar(&s.Namespace, "namespace", c.Namespace, "Namespace where devops system is in.")
 	fs.StringVar(&s.WorkerNamespace, "worker-namespace", c.WorkerNamespace, "Namespace where Jenkins agent workers are in.")
+	fs.DurationVar(&s.ReloadCasCPeriod, "reload-casc-period", c.ReloadCasCPeriod,
```

Comment:
`period` is incorrect. it should be `delay`.

Reply:
My mistake. I misunderstand some fields related period in [Kubernetes type](https://github.com/kubernetes/kubelet/blob/c4c83cb3f378ed877f96f12d2e874d9a7888e521/config/v1beta1/types.go#L978-L983).